packages/zoe/test/unitTests/contracts/loan/test-addCollateral.js

```diff
@@ -1,8 +1,7 @@
 // @ts-nocheck
```

Comment:
ha, well, a few reasons:
- out of scope for PR
- not a priority before MN-1 to add types to code we're not working in
- types in tests are only for the test writer since they don't provide any additional confidence in production

Reply:
It is interesting to see how our coding standards for test code increasingly differs from our coding standards for production code. Another is that we should not impose our top-level-await rule on test code. Will we maintain divergent lint settings?

In any case, for this PR no change suggested. Resolving.

Reply:
> our coding standards for test code increasingly differs from our coding standards for production code

I contend this is appropriate.

> Will we maintain divergent lint settings?

It's managed by a rule tightening for production code:
https://github.com/Agoric/agoric-sdk/blob/9410a8500f6a30906c9e3359ece0e5d8dfd965ad/.eslintrc.cjs#L73-L91